### DISCUSSION TITLE
Ability to organize daily notes in folders

### DISCUSSION THREAD

**Discussion by igtsimoes** [notaapp/talk#390] [Feature Requests]
I wanted to organize my daily notes in folders and not lose the search or backlinking capabilities. 

In this picture, I have the daily notes as they are - all into the daily notes directory, without subfolders. 

<img width="212" alt="image" src="https://user-images.githubusercontent.com/64813530/230746626-633f785b-8212-4303-9d09-83fad7671f2b.png">

But, if I create a "01-Jan" subfolder and move all my January/2023 notes there, this is what happens: 

<img width="247" alt="image" src="https://user-images.githubusercontent.com/64813530/230746607-5f6c7a5e-7b77-4afb-a63c-5c044597d9f3.png">

Daily notes might become quite big, 365 a year 😄 . Organizing into folders like year/month makes more sense to me. 

What do you think?

**Comment by astoilkov** [comment]
Yeah, I agree. This is a good idea. It's easy to fix the search. However, I'm not sure how we should approach adding the daily note to the appropriate folder. Any ideas? Have you seen this somewhere?

**Comment by igtsimoes** [comment]
@astoilkov I think Nota should only handle the search. 

The process of organizing the daily folder should be up to each user. Some may prefer to organize by year, others by weeks, months, etc. 

What I think Nota should handle is recognizing that files inside the daily notes folder are actually the daily notes. The subfolders would be only for organizing by user preference.

**Reply by astoilkov** [reply]
Aha, got it. I think I can do the recognition for the next release. Writing this down. Thanks!

**Reply by igtsimoes** [reply]
Perfect! Thank you!! 

@astoilkov When are you guys planning the next release?

**Reply by astoilkov** [reply]
We're planning it for the end of May.
